apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx

```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
 mode: draft
 route: /patterns/placement-and-order/
 ---
+## Guidelines
+In order to place yout objects and component the right way, you have always to keep in mind the ready pattern of your user. 2 types of patterns can be encountered: F and Z.
```

Comment:
A few mistakes here, re-write it to say: 
In order to place your objects and components correctly, you must always keep in mind the reading pattern of your user. Two types of patterns can be encountered: F and Z.

Reply:
Updated